src/main/scala/cromwell/binding/types/WdlMapType.scala

```diff
@@ -0,0 +1,7 @@
+package cromwell.binding.types
+
+case class WdlMapType(keyType: WdlPrimitiveType, valueType: WdlType) extends WdlType {
+  val toWdlString: String = s"Map[${keyType.toWdlString}, ${valueType.toWdlString}]"
+  override protected def coercion = ???
+  override def fromRawString(s: String) = ???
```

Comment:
We will, I just didn't implement it yet.  I think I just added this for completeness but I should probably remove it and make it part of another PR.